inc/shortcodes/wikipublisher/class-glyphs.php

```diff
@@ -78,7 +77,7 @@ public function langShortcode( $atts, $content = null ) {
 		if ( empty( $content ) || empty( $a['lang'] ) || ! in_array( $a['lang'], $this->supported_languages, true ) ) {
 			// We don't support this language
 			$_error = "*** ERROR: Unsupported pb_language attribute: {$a['lang']} -- ";
-			$_error .= 'Valid choices, based on ISO-639-3, are: ' . implode( $this->supported_languages, ', ' ) . ') ***';
+			$_error .= 'Valid choices, based on ISO-639-3, are: ' . implode( ', ', $this->supported_languages, ) . ') ***';
```

Comment:
Switches order of parameters for implode function as required in PHP 7.4 (https://www.php.net/manual/en/function.implode.php)